vite.config-es.ts

```diff
@@ -12,7 +12,7 @@ export default defineConfig({
     lib: {
       entry: resolve(__dirname, 'src/leaflet-maptilersdk.ts'),
       name: 'leafletmaptilersdk',
-      fileName: () => "leaflet-maptilersdk.min.js",
+      fileName: (format, entryName) => "leaflet-maptilersdk.js",
```

Comment:
@petrsloup tbh I have no idea why and when it was changed - copy'n'paste error I guess 😞 Probably my fault.